pallets/task/src/lib.rs

```diff
@@ -432,10 +507,44 @@ pub mod pallet {
 			Ok(())
 		}
 
+		// Task can be rejected by the creator, which places the task back into progress.
+		pub fn reject_completed_task(task_initiator: &T::AccountId, task_id: &T::Hash) -> Result<(), DispatchError> {
+			
+			// Check if task exists
+			let mut task = Self::tasks(&task_id).ok_or(<Error<T>>::TaskNotExist)?;
+
+			// Check if the owner is the one who created task
```

Comment:
should we check here that task status is InProgress? or a Task which is in Created state can be rejected too, however which does not make sense to me.

Reply:
Good!. Added a check.